chromium_src/chrome/browser/ui/views/toolbar/side_panel_toolbar_button.h

```diff
@@ -16,13 +16,21 @@
 
 #undef SidePanelToolbarButton
 
+class Browser;
+
 class SidePanelToolbarButton : public SidePanelToolbarButton_ChromiumImpl {
  public:
   explicit SidePanelToolbarButton(Browser* browser);
 
   SidePanelToolbarButton(const SidePanelToolbarButton&) = delete;
   SidePanelToolbarButton& operator=(const SidePanelToolbarButton&) = delete;
   ~SidePanelToolbarButton() override;
+
+ private:
+  void UpdateButtonImage();
+
+  const raw_ptr<Browser> browser_ = nullptr;
```

Comment:
I guess doing this is easier than exposing the private variable in `SidePanelToolbarButton_ChromiumImpl`. No strong feelings here.

Reply:
Thanks for review! I realized that we don't need to have `browser_` :)
Deleted!